### DISCUSSION TITLE
ADGuard Home: migration solution from Raspian32 to other Linux 64bit

### DISCUSSION THREAD

**Discussion by AntonioScorza** [AdguardTeam/AdGuardHome#3776] [Q&A]
Hi All,
I searched in the discussions but I found nothing.
How I can move the ADGuard Home solution from Raspbian 32bit to other Linux 64bit?
For migration, I read that I can copy only the AdGuardHome.yaml file.
I tried but I loose the statistics.
Do you have the full instruction to migrate all?

Regards.
Antonio

**Comment by ainar-g** [comment]
You should copy the configuration file, the `data` directory, as well as the `leases.db` file, if you use the DHCP server.

We will document both that as well as a better guide for switching between channels when we finally have the time.
